react-carousel/my-app/src/Dot.tsx

```diff
@@ -0,0 +1,18 @@
+import { FaCircle } from 'react-icons/fa';
+
+type Prop = {
+  count: number;
+  onDotClick: (index: number) => void;
+};
+
+export function Dot({ count, onDotClick }: Prop) {
+  function renderButton() {
+    const buttons = [];
+    for (let i = 0; i < count; i++) {
+      buttons.push(<FaCircle key={i} onClick={() => onDotClick(i)} />);
```

Comment:
You should render a filled-in circle for the dot that represents the current image.